### ISSUE TITLE
Printer objects in the printer Class

### ISSUE THREAD

**Issue by michaelrsweet** [apple/cups#596]
Version: -feature
CUPS.org User: angelb.bugarin

Classes are great for grouping printers with specific type/function;
ie, SAP, Accoutning, Engr, etc.

Currently, when you access a printer Class, you get a listing of
queue names that belongs to the Class.

Will there be a patch for CUPS so that when you access the printer
Class, you are presented with the actual printer objects like the
Printers function instead of just names? This will be usefull
specially when dealing with hundreds of printers. From the Class, you
can then perform the normal printer administration. Currently, I have
to search a printer from the list provided by the Printers function.

Simply put, show a list of printers, not just names, and the admin controls when accessing a printer class.

Thanks!
Angel
